app/src/main/res/layout/activity_thread_layout.xml

```diff
@@ -35,44 +35,47 @@
         </com.kickstarter.ui.toolbars.KSToolbar>
     </com.google.android.material.appbar.AppBarLayout>
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:focusable="true"
-        tools:context="com.kickstarter.ui.activities.ThreadActivity"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior"
-        android:orientation="vertical">
+   <androidx.constraintlayout.widget.ConstraintLayout
+       android:layout_width="match_parent"
+       app:layout_behavior="@string/appbar_scrolling_view_behavior"
+       android:layout_height="match_parent">
 
-        <com.kickstarter.ui.views.CommentCard
-            android:id="@+id/comments_card_view"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+       <androidx.core.widget.NestedScrollView
+           android:layout_width="match_parent"
+           android:layout_height="0dp"
+           android:fillViewport="true"
+           app:layout_constraintBottom_toTopOf="@+id/reply_composer"
+           app:layout_constraintTop_toTopOf="parent"
+           tools:layout_editor_absoluteX="0dp">
+
+          <androidx.constraintlayout.widget.ConstraintLayout
+              android:layout_width="match_parent"
+              android:layout_height="match_parent">
+
+              <com.kickstarter.ui.views.CommentCard
+                  app:layout_constraintTop_toTopOf="parent"
+                  android:id="@+id/comments_card_view"
+                  android:layout_width="match_parent"
+                  android:layout_height="wrap_content" />
 
-        <androidx.constraintlayout.widget.Guideline
-            android:id="@+id/left_guideline"
-            app:layout_constraintGuide_begin="@dimen/grid_5"
-            android:orientation="vertical"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"/>
+              <androidx.recyclerview.widget.RecyclerView
```

Comment:
Add constraints for this two elements. When there is only one reply the UI shows like this: 
![Screen Shot 2021-06-22 at 1 59 22 PM](https://user-images.githubusercontent.com/4083656/122999252-e0667980-d362-11eb-87da-72a53c85b29a.png)

- Also check please the separator between items seems to not been there for the recyclerView items 
![Screen Shot 2021-06-22 at 1 59 03 PM](https://user-images.githubusercontent.com/4083656/122999329-fc6a1b00-d362-11eb-8c87-71dba5441b0e.png)